src/components/student/ScheduleView.js

```diff
@@ -9,14 +14,130 @@ import React, {useState} from 'react';
 // issue a DELETE with URL /enrollment/{enrollmentId}
 
 const ScheduleView = (props) => {
-    
-   
-    return(
-        < > 
-            <h3>Not implemented</h3>
-        </ >
-    );
-
-}
+
+    const headers = ['SecNo', 'CourseId', 'SecId',  'Year', 'Semester', 'Building', 'Room', 'Times', '', ''];
+
+    const [search, setSearch] = useState({
+        scheduleId: '',
+        year: '',
+        semester: ''
+    });
+    const [message, setMessage] = useState('');
+    const [schedules, setSchedules] = useState([]);
+
+    const fetchSections = async () => {
+            if (search.courseId==='' || search.year==='' || search.semester==='' ) {
```

Comment:
We might want to check that year is an int and matches a year pattern. for example 4 digits. Also, we may want to check the semester matches a semester "Sring", "Fall", etc. We can display a message that hints what the right input should be.

Reply:
@nickleus27 Just tested the file and had to remove CourseId, I agree only need to lookup by semester and year. Issue has been resolved and pushed the changes.

Reply:
@numbee12 I forgot this was a GET request and not a PUT or POST. So it doesn't really matter so much to validate the input values because it will just not return anything with malformed input. But maybe, checking that year is integer is still worth while because I think year is an int type.